articles/tokens.md

```diff
@@ -0,0 +1,484 @@
+https://github.com/auth0/auth0.js#delegation-token-request
+
+###How to get a delegation token
+The `id_token` for an authenticated user can be used with the /delegation endpoint to request a delegation token for a particular target.  The target can be either another application/API registered in Auth0 or an application Addon configured in Auth0.  The Addons for which this can be done are those that are not SAML or WS-Fed Addons and the Addon must be configured in Auth0 with secrets obtained from the Addon service, such as Firebase.  Instructions for setting up the secrets are available from the Addon configuration page for each Addon.  The secrets are used to sign the delegation token so that the Addon API can validate and trust the token.
+
+Information on the delegation endpoint is at:
+https://auth0.com/docs/auth-api#delegated
+
+###How to control contents of a delegation token
+The delegation endpoint allows the setting of several parameters which will govern the contents of the delegation token, including the target, the scope, the API to be called and an additional free-form area for additional parameters.
+
+See the delegation endpoint in the Authentication API for more information.
+
+###Validity
+For customer application APIs registered in Auth0  - the validity of a delegation token issued for that target is governed by the setting for each application API in Apps/APIs -> Settings for “JWT expiration”.
+
+For APIs registered as Addons in Auth0, the validity period of the token will vary by individual Addon.  The documentation available from the provider of any Addon API should be consulted for further information on tokens and expirations.
+
+
+###Renewal of a delegation token
+When a delegation token expires, the delegation endpoint can be used to obtain a token.
+
+###Terminating a delegation token
+The ability to revoke a delegation token will vary by individual Addon.  The documentation available from the provider of any Addon API should be consulted for further information on whether a token can be revoked and if so, how to do it.
+
+The delegation tokens for customer APIs registered in Auth0 cannot be revoked.  A best practice, therefore, is to set the JWT expiration duration to a relatively short value.
+
+###Uses
+A delegation token should be obtained and used when a client program needs to call the API of either a) another application/API registered in Auth0 or b) the API for an Application Addon, such as Firebase or SAP, registered and configured in Auth0, in the same tenant as the calling program.
+https://github.com/auth0/auth0.js#delegation-token-request
+
+https://auth0.com/docs/libraries/auth0js#12
+
+###Best Practices
+For granularity of access control, good to set up different applications/APIs with different secrets, so that a delegation token can be issued uniquely for each application/API.
+
+Tokens should be issued with a short timeframe for expiration, where configurable.
+
+
+## The Auth0 APIv2 access token
```

Comment:
@twistedstream Hi, I like the 'api' part of the name, but not sure we need the 'jwt' in it.  None of the other names allude to internal format details of the token.  I think it would be more helpful to clarify in the name which api it is for because there are several apis, each of which requires a different token.  It would eliminate a lot of confusion if
a) we use the same name (fully qualified) consistently throughout our docs for each type of token (e.g. don't just call it 'the token')
b) each token is named to indicate what it is and/or what you use it for. - auth0 api token, IDP api token, 3rd party app API token, etc.  
I'm not strongly against having jwt in the name, but I think the above 2 things are more important in helping people understand all these tokens and which ones they need.